app/assets/stylesheets/devise/login.scss

```
@@ -0,0 +1,23 @@
+.login_exterior_container
+{
+  margin-top: 10%;
+  background-color: #ebf3fa; 
+  border-radius: 2%; 
+  max-width: 500px; 
+  border: solid #c3cce8
+}
```

```suggestion
.login_exterior_container {
  margin-top: 10%;
  background-color: #ebf3fa; 
  border-radius: 2%; 
  max-width: 500px; 
  border: solid #c3cce8
}
```

Change the rest as well, adding the '{' in the first line